src/plots/gl3d/scene.js

```diff
@@ -31,6 +30,32 @@ var createAxesOptions = require('./layout/convert');
 var createSpikeOptions = require('./layout/spikes');
 var computeTickMarks = require('./layout/tick_marks');
 
+var isMobile = require('is-mobile');
+var tablet = isTablet();
+
+function isTablet() {
+    var navigator = window.navigator;
+    var userAgent = navigator.userAgent;
+
+    var result = isMobile({
+        ua: userAgent,
+        tablet: true
+    });
+
+    // handle iPad pro or iPad with iOs 13 using Safari
+    // see https://github.com/plotly/plotly.js/issues/4502
+    if(
+        result === false &&
+        userAgent.indexOf('Macintosh') !== -1 &&
+        userAgent.indexOf('Safari') !== -1 &&
+        navigator.maxTouchPoints > 1
```

Comment:
How did you find this solution?

Reply:
Inspired by info on https://stackoverflow.com/questions/58019463/how-to-detect-device-name-in-safari-on-ios-13-while-it-doesnt-show-the-correct